gold-digger/src/DiggerModel.js

```diff
@@ -14,7 +14,7 @@ class DiggerModel{
         this.genres = [];   // String values
         this.includedArtists = [];  // Spotify ID
         this.excludedArtists = [];
-        this.prevPlaylists = prevPlaylists; // [{playlist:, tracks: }, ...]
+        this.prevPlaylists = prevPlaylists; // [{playlist:, key: }, ...]
```

Comment:
Might conflict with what I have but it should be easy to fix.

Reply:
Great! This is the change we discussed during the meeting, where we don't want to save all tracks for all previously generated lists but rather only the firebase key